src/api/app/jobs/bs_request_action_webui_infos_job.rb

```diff
@@ -48,8 +49,18 @@ def for_a_superseded_request(superseded_request, bs_request_action, source_packa
       BsRequestAction::Differ::ForSource.new(
         bs_request_action: bs_request_action,
         source_package_names: source_package_names,
-        options: { superseded_bs_request_action: superseded_bs_request_action }
+        options: query.merge({ superseded_bs_request_action: superseded_bs_request_action })
       ).perform
     end
   end
+
+  def query
```

Comment:
Shouldn't `skip_access_check` also be in here? Or alternatively, shouldn't all of this be on line 15?


And `query` is not such a nice name. It's not a query right?

Reply:
I've removed this method. It's not needed anymore.